standard/info/wikis/counterstrike/info.lua

```diff
@@ -10,19 +10,86 @@ return {
 	startYear = 2000,
 	wikiName = 'counterstrike',
 	name = 'Counter-Strike',
+	games = {
+		cs = {
```

Comment:
0422ee8f967f78215868a3c6618c378309619a2c
don't know if rath will like it style wise but it should work
if you could test that it would be apreciated
i need some cookies after doing this chore

Reply:
Prefer just one name for it. Looking at the CS wiki, cs16 makes more sense. Template:CS is the one one I could fine using `cs` instead of `cs16`, and it redirects to a `cs16` page.

Reply:
if we change it to cs16 it will work as the toIdentifier function i have in the Games module (pending PR for after this PR) also checks the lowercased abbreviations